charts/umbrella/values.yaml

```diff
@@ -790,6 +790,22 @@ tx-data-provider:
     ingress:
       enabled: false
 
+semantic-hub:
```

Comment:
could you please explicitly disable persistence?

Reply:
I disabled semantic hub keycloak postgres persistence. As for the graphdb, it can't be disabled by design, if you look at the manifests for the semantic-hub [here](https://github.com/eclipse-tractusx/sldt-semantic-hub/blob/f522a0aca7c50e6a7be4b98d0aa361aab3394709/charts/semantic-hub/templates/graphdb/graphdb-deployment.yaml#L66). We either disable graphdb entirely or have the PVC generated.